github/git_commits.go

```diff
@@ -68,8 +68,7 @@ func (s *GitService) GetCommit(ctx context.Context, owner string, repo string, s
 		return nil, nil, err
 	}
 
-	// TODO: remove custom Accept headers when APIs fully launch.
-	req.Header.Set("Accept", mediaTypeGitSigningPreview)
+	req.Header.Set("Accept", mediaTypeV3)
```

Comment:
Thank you, @palash25.

Please note that here and everywhere else in this PR, `mediaTypeV3` does not need to be explicitly added, as that is handled in `github.go`. Please address this and I'll review again.

Reply:
Removed it in the latest commit @gmlewis